distributed/worker.py

```diff
@@ -635,6 +635,15 @@ def __init__(
             "offload": utils._offload_executor,
             "actor": ThreadPoolExecutor(1, thread_name_prefix="Dask-Actor-Threads"),
         }
+        try:
+            import pynvml
+        except ImportError:
+            pass
+        else:
+            if pynvml.nvmlDeviceGetCount():
```

Comment:
We probably want to use `device_get_count` from `distributed/diagnostics/nvml.py` here

https://github.com/dask/distributed/blob/d1cd7fa0e69e2f60a1938b83786c5b3745af4811/distributed/diagnostics/nvml.py#L33-L38

cc @pentschev

Reply:
I'm happy to gift this PR to anyone else (@pentschev seems like a natural recipient).  Mostly I wanted to make the issue in #5084 more real.